src/main/resources/db/migration/V3__modify_users_table.sql

```diff
@@ -1,6 +1,9 @@
 ALTER TABLE users
     ADD password VARCHAR(60);
 
+ALTER TABLE users
+    ADD user_type INTEGER;
```

Comment:
ez miért is Integer?  ez így véleményem szerint nem lesz jó, de mivel új adatbézisunk lesz, ezt hagyd a fenébe

Reply:
mivel először VARCHARnak adtam meg de szólt a hibaüzenet, hogy ez bizony INTEGER és mikor megváltoztattam szépen lement.

Reply:
hmm, ez izgalmas lesz amikor újrahúzom az adatbázis :D De megjegyzem.